docs/modules/ROOT/pages/usage/lsp.adoc

```diff
@@ -0,0 +1,47 @@
+= LSP (Language Server Protocol)
+
+NOTE: The built-in language server was introduced in RuboCop 1.53.
+
+https://microsoft.github.io/language-server-protocol/[The Language Server Protocol] is the modern standard for providing cross-editor support for various programming languages.
+
+This feature enables extremely fast interactions through the LSP.
+
+== Run as a Language Server
+
+Run `rubocop --lsp` command from LSP client.
```

Comment:
I'm wondering if a name more consistent with the existing commands (e..g `--start-lsp-server`) is not idea as well. We can keep `--lsp` as a shorter alias.

Reply:
I'm concerned that LSP client is the trigger that reconnects or shuts down the LSP server. For Elgot, use `M-x elgot-reconnection` or `M-x elgot-shutdown` to reconnect or shutdown. In other words, the `rubocop` command that the user recognizes in the LSP setting may be enough with `--lsp` option.
How about just providing `--lsp` option to start with? So `--start-lsp-server` will be added later when really needed for user.

Reply:
Got it. If that's the only flag the users will ever need we can stick with it.